MediaGalleryRenditions/etc/module.xml

```diff
@@ -6,5 +6,11 @@
  */
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Magento_MediaGalleryRenditions"/>
+    <module name="Magento_MediaGalleryRenditions">
+        <sequence>
```

Comment:
I don't think any updates to sequence are necessary, please update `composer.json` to identify dependencies